models/dex/dex_schema.yml

```diff
@@ -53,27 +53,39 @@ models:
       - &token_bought_address
         name: token_bought_address
         description: "Contract address of the token bought"
+        tests:
+          - assert_evm_address
```

Comment:
this is gonna be such a useful generic test.

Reply:
oh yeah this is great!!!